mpxe/sims/adt7476a.py

```diff
@@ -0,0 +1,25 @@
+from mpxe.protogen import stores_pb2
+from mpxe.protogen import adt7476a_pb2
+
+from mpxe.sims import sim
+
+class Adt7476a(sim.Sim):
+    def handle_update(self, pm, proj):
+        stores  = proj.stores
+        # for some reason there's a GPIO store initializes before
```

Comment:
this is expected behaviour so probably don't need a comment for it, also should remove the comment `# print(stores)`. Ideally don't use first person in comments since this code will probably be with the team longer than you'll be